src/shim/sdk/DocumentSDK.ts

```diff
@@ -129,7 +129,7 @@ export function decrypt(documentID: string, documentData: Uint8Array) {
         type: "DOCUMENT_DECRYPT",
         message: {
             documentID,
-            documentData,
+            documentData: documentData.slice(),
```

Comment:
https://jsperf.com/new-array-vs-splice-vs-slice/151

Looks like the right one! `slice(0)` is a bit faster for me on FF, but `slice()` is only barely slower. In Chrome `slice()` is significantly the fastest.